README.md

```diff
@@ -16,22 +16,19 @@
 [travis]: https://travis-ci.org/mcu-tools/mcuboot
 [license]: https://github.com/mcu-tools/mcuboot/blob/main/LICENSE
 
-This is mcuboot version 1.8.0-rc1
+This is MCUboot version 1.8.0-rc1
 
-MCUboot is a secure bootloader for 32-bit MCUs. The goal of MCUboot is to
-define a common infrastructure for the bootloader, system flash layout on
-microcontroller systems, and to provide a secure bootloader that enables
-simple software upgrades.
+MCUboot is a secure bootloader for 32-bit MCUs.
+The goal of MCUboot is to define a common infrastructure for the bootloader, system flash layout on microcontroller systems, and to provide a secure bootloader that enables simple software upgrades.
 
-MCUboot is operating system and hardware independent and relies on
-hardware porting layers from the operating. Currently, mcuboot works
-with both the Apache Mynewt and Zephyr operating systems, but more
-ports are planned in the future. RIOT is currently supported as a boot
-target with a complete port planned.
+MCUboot is operating system and hardware independent and relies on hardware porting layers from the operating system.
+Currently, MCUboot works with both the Apache Mynewt and Zephyr operating systems, but more ports are planned in the future.
```

Comment:
There are Apache NuttX RTOS and Mbed-OS ports.

Reply:
Thanks. I was wondering about that, as it looked like outdated info, contradicting other parts of the docs. However, what about RIOT OS? Is the implementation still "stub" (as mentioned two lines below) or it should be listed there as well?

Reply:
RIOT OS port efforts has no visible progress since years - mention of it is worthless.
^^ @d3zd3z Pleas correct me If I'm wrong.

We can kept the sentence regard planed ports - but in reality this is something which might be added by the community an usually is contributed by interested party (on they will). We don't have well defined plan to add more ports but we will accept any new port once it is good enough.

Reply:
fixed